Example/Source/Bricks/Custom/SegmentHeaderBrick.swift

```diff
@@ -40,6 +42,12 @@ class SegmentHeaderBrickCell: BrickCell, Bricklike {
 protocol SegmentHeaderBrickDataSource: class {
     var titles: [String] { get }
     var selectedSegmentIndex: Int { get }
+
+    func configure(cell: SegmentHeaderBrickCell)
+}
+
+extension SegmentHeaderBrickDataSource {
+    func configure(cell: SegmentHeaderBrickCell) {/*Optional*/}
```

Comment:
This is an old brick. Ideally, we should use `GenericBrick<UISegementedControl>`...